app/(main)/home/MidSectionText.tsx

```diff
@@ -0,0 +1,7 @@
+export default function MidSectionText() {
+  return (
+    <h1 className="justify-center text-center text-3xl font-bold font-bold leading-[45px] text-[#364071]">
```

Comment:
- Can remove `justify-center` as this is not a flex container. 
- Extra `font-bold`
- Should we make a color in tailwind config for `#364071` ? I think it's used in multiple places.